apps/engine/lib/execution/src/runner/javascript/apache-arrow-bundle.js

```diff
@@ -3,7 +3,7 @@ export var arrow;
   // webpackBootstrap
   /******/ "use strict";
   /******/ var __webpack_modules__ = {
-    /***/ 717: /***/ (
+    /***/ 35: /***/ (
```

Comment:
Result of re-running instructions in `apps/engine/apache-arrow-js-bundle/README.md`